client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -33,15 +33,16 @@ export default React.createClass( {
 
 	render() {
 		const { site, redirect, linkDisplay } = this.props;
-		const buttonElement = linkDisplay ? 'button' : Button;
```

Comment:
Ok, I think I see how that works now. That said, the [`.is-link` styles](https://github.com/Automattic/wp-calypso/blob/3e64e7284655bdc5480799b68679c90845556ee4/client/components/button/style.scss#L260-L276) are part of the `<Button />` component itself, so it feels odd to apply the modifier and `button` classes from a parent component. Further, they're classified under a "deprecated styles" heading. I suppose the alternative would be to render the base `button` HTML element since it appears [they're similarly reset to link styling](https://github.com/Automattic/wp-calypso/blob/3e64e7284655bdc5480799b68679c90845556ee4/client/components/button/style.scss#L5-L15), to try to render the `<Button />` with a `href` prop, or undeprecate the `.is-link` styling and allow an additional prop to be accepted by `<Button />` to control it (cc @rickybanister for clarification re: #1370).